packages/manager/apps/telecom/src/app/telecom/pack/slots/hostedEmail/detail/detail.controller.js

```diff
@@ -0,0 +1,241 @@
+    this.accountDeleting = false;
+    this.loading = true;
+
+    this.passwordRules = [
+      {
+        id: 'length',
+        caption: this.$translate.instant(
+          'hosted_email_detail_change_password_rule_size',
+        ),
+        validator(str) {
+          return str && str.length > 7 && str.length < 21;
+        },
+      },
+      {
+        id: 'specialChar',
+        caption: this.$translate.instant(
+          'hosted_email_detail_change_password_rule_special',
+          { list: '#{}()[]-|@=*+/!:;' },
+        ),
+        validator: /^[\w~"#'{}(\\)[\]\-|\\^@=*+/!:;.,?<>%*µÀÁÂÃÄÅàáâãäåÒÓÔÕÖØòóôõöøÈÉÊËèéêëÇçÌÍÎÏìíîïÙÚÛÜùúûüÿÑñ]+$/,
+        immediateWarning: true,
+      },
+    ];
+
+    return this.$q
+      .all([this.loadConfiguration(), this.getAccount(), this.loadPackInfo()])
+      .finally(() => {
+        this.loading = false;
+      });
+  }
+
+  toHuman(valueParam) {
+    let value = valueParam;
+    const orig = value;
+    const units = [
+      'hosted_email_detail_byte',
+      'hosted_email_detail_Kb',
+      'hosted_email_detail_Mb',
+      'hosted_email_detail_Gb',
+      'hosted_email_detail_Tb',
+    ];
+    let index = 0;
+    while (value / 1024 > 1 && index < units.length) {
+      value /= 1024;
+      index += 1;
+    }
+    return {
+      raw: orig,
+      human: Math.round(value * 10) / 10,
+      unit: this.$translate.instant(units[index]),
+    };
+  }
+
+  /**
+   * Load pack information
+   * @return {Promise}
+   */
+  loadPackInfo() {
+    return this.OvhApiPackXdsl.Aapi()
+      .get({
+        packId: this.packName,
+      })
+      .$promise.then((packInfo) => {
+        this.pack = packInfo.general;
+        return this.pack;
+      });
+  }
+
+  /**
+   * Load detail of the hosted email account
+   * @return {Promise}
+   */
+  loadConfiguration() {
+    return this.OvhApiPackXdslHostedEmail.v6()
+      .getConfig({
+        packId: this.packName,
+        domain: this.serviceName,
+      })
+      .$promise.then((config) => {
+        this.configuration = config;
+
+        this.configuration.services = filter(
+          map(this.configuration.services, (service, type) =>
+            isObject(service) ? assignIn({ type }, service) : false,
+          ),
+          (service) => isObject(service),
+        );
+        this.configuration.status = {
+          value: this.configuration.status,
+          label: this.configuration.status
+            ? this.$translate.instant('hosted_email_detail_active')
+            : this.$translate.instant('hosted_email_detail_inactive'),
+          icon: this.configuration.status
+            ? 'ovh-font ovh-font-filled-check text-success'
+            : 'ovh-font ovh-font-filled-error text-warning',
+        };
+        return this.configuration;
+      })
+      .catch((err) => {
+        this.TucToast.error(
+          this.$translate.instant('hosted_email_detail_loading_error'),
+        );
+        return this.$q.reject(err);
```

Comment:
Hmm you're not catching the error in your `$q.all` call, so I dont think this return is needed.